src/CachePoolTest.php

```diff
@@ -190,8 +198,23 @@ public function testGetItemsEmpty()
         }
 
         $items = $this->cache->getItems([]);
+        if (is_object($items)) {
+            $this->assertInstanceOf(
+                'Traversable',
+                $items,
+                'A call to getItems with an empty array must always return an array or \Traversable.'
+            );
+
+            $items = iterator_to_array($items);
+        } else {
+            $this->assertInternalType(
+                'array',
+                $items,
+                'A call to getItems with an empty array must always return an array or \Traversable.'
+            );
+        }
+
         $this->assertCount(0, $items);
-        $this->assertTrue(is_array($items) || $items instanceof \Traversable, 'A call to getItems with an empty array must always return an array or \Traversable.');
```

Comment:
this one was fine to me, you only need to add a `foreach` after it and `$count++` like you above, isn't it?